style_editor.html

```diff
@@ -22,19 +22,53 @@
 * {
   box-sizing: border-box;
 }
+.page-left {
+  vertical-align: top;
+  width: 50%;
+}
+.page-right {
+  vertical-align: top;
+ }
 
-/* Expand and Layerize buttons */
-.middle-btn-container .button-off,
-.middle-btn-container .button-off:hover {
+/* This keeps the More Effects menu and
+the Do Selected Effect button together */
+.more-menu-container {
+  white-space: nowrap;
+}
+
+/* Do Selected Effect, Expand, and Layerize */
+.page-left .button-off,
+.page-left .button-off:hover {
   background-color: lightgray;
   color: darkgray;
   border: 2px solid lightgray;
   padding: 2px;
 }
-.middle-btn-container .button-on {
+.page-left .button-on {
+  padding: 2px;
+  border: 2px solid dodgerblue;
+}
+.page-left .button-on:hover {
+  background-color: #ddd;
   padding: 2px;
   border: 2px solid dodgerblue;
 }
+.page-left .button-on:active {
+  background-color: dodgerblue;
+}
+
+/* Lockup and More Effects Drop Downs */
```

Comment:
Organizing. grouped and moved these to be in same order as they appear on the page.